dashboard/modules/resource/tests/test_resource.py

```diff
@@ -0,0 +1,244 @@
+import time
+import ray
+import requests
+import pytest
+
+from ray.dashboard.tests.conftest import *  # noqa
+from ray._private.test_utils import (
+    format_web_url,
+    wait_until_server_available,
+)
+
+
+@ray.remote(num_cpus=1, memory=1000, resources={"custom_a": 1})
```

Comment:
Define these within the test. Otherwise, it will implicitly start ray in the background.

Reply:
Hmm ok. Its just the function names become slightly mangled within a test.